Makefile

```diff
@@ -157,6 +157,12 @@ test/common:
 	git checkout --force FETCH_HEAD -- test/common
 	git reset test/common
 
+patternfly:
+	git fetch --depth=1 https://github.com/cockpit-project/cockpit.git 227
```

Comment:
If you want to test this early, you can also fetch the origin without depth and check out a SHA.